solutions/usaco-988.mdx

```diff
@@ -81,7 +84,7 @@ int main() {
 **Time Complexity:** $\mathcal{O}(N^3)$
 
 <Info title="Time Complexity">
-Since the `in` operator in Python takes $\mathcal{O}(N)$ time, this solution is slower than the C++ version.
+Since the `in` operator in Python takes $\mathcal{O}(N)$ time, this solution is $\mathcal{O}(N^3)$.
```

Comment:
i think this is fine, the actual time complexity is right above it, and the info is to let you know that another language would be faster? but up to you